### PR TITLE
Initial endpoint

### PR DESCRIPTION
**Pull request by praveensree**

### CODE REVIEW

**Review by akshayverma91 — CHANGES_REQUESTED** [review]
This is not default API structure that we have discussed before starting on stories
I have asked to create Post and comment empty api structure

**Comment by akshayverma91** [comment]
@praveensree change name post to SocialPost I'll make more sense to have Post of what domain so it should be like SocialPost

File: Server/Controllers/SocialPostController.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+namespace Server.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class SocialPostController : ControllerBase
+    {
+        // GET: api/SocialPost
+        [HttpGet]
+        public IEnumerable<string> GetAllPosts()
```
Comment (akshayverma91) [inline Server/Controllers/SocialPostController.cs]: It should be GetAllPost

File: Server/Controllers/SocialPostController.cs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+        [HttpPut("Heart/{id}")]
+        public void UpdateHeart([FromBody] string value, int id)
+        {
+            //TODO: Have to write logic here
+        }
+
+        // DELETE api/SocialPost/5
+        [HttpDelete("{id}")]
+        public void DeletePostById(int id)
```
Comment (akshayverma91) [inline Server/Controllers/SocialPostController.cs:59]: We don't need to delete any post please confirm requirement once again, even avinash can help you out in grooming of requirement

File: Server/Controllers/CommentsController.cs [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+        // PUT api/<CommentsController>/5
+        [HttpPut("{id}")]
+        public void UpdateCommentById([FromBody] string value, int id)
+        {
+        }
+
+        // DELETE api/<CommentsController>/5
+        [HttpDelete("{id}")]
+        public void DeleteCommentById(int id)
```
Comment (akshayverma91) [inline Server/Controllers/CommentsController.cs:43]: NO need to delete a comment

File: Server/Controllers/SocialPostController.cs (hunk 2 above)
Comment (praveensree) [reply Server/Controllers/SocialPostController.cs]: Removed Delete method for SocialPost

File: Server/Controllers/CommentsController.cs (hunk 3 above)
Comment (praveensree) [reply Server/Controllers/CommentsController.cs]: Removed the Delete method from Comments

File: Server/Controllers/SocialPostController.cs (hunk 1 above)
Comment (praveensree) [reply Server/Controllers/SocialPostController.cs]: Changed the Method Name to GetAllPost

**Review by akshayverma91 — APPROVED** [review]
looks good

**Pull request merged**